README.md

```diff
@@ -210,3 +210,12 @@ OAUTH2_JWKS_URL
 OAUTH2_AUDIENCE
 OAUTH2_ALGORITHMS
 SECRET_MANAGER
+
+## Running tests
+
+For developers to run a test use the following,
+
+```bash
+pipenv shell
+APP_ENV='TEST' pytest
```

Comment:
Use `pytest.ini` to set env variables. https://docs.pytest.org/en/latest/reference.html#configuration-options


So...in the root of this repo you can find an (empty) `pytest.ini`. In it, add:

```
[pytest]
APP_ENV = 'TEST'
```

Reply:
All right. I tried this out, and now one of my tests fail.

```
E       AssertionError:
E       expected: 400 to equal 201

tests/test_startup.py:26: AssertionError
```

That's odd.